src/GlobalStyles.js

```diff
@@ -18,27 +18,27 @@ main {
 
 
 h1 {
-    font-size: 2.5em;
+    font-size: 29px;
```

Comment:
wouldn't it be better in the old way to config the headings-size in em and not in px.
Do that you can define a font-size in the whole body and you don't have to change all of the headings-sizes afterwards manually, when you change font-size for the body, later?

Reply:
> wouldn't it be better in the old way to config the headings-size in em and not in px.
> Do that you can define a font-size in the whole body and you don't have to change all of the headings-sizes afterwards manually, when you change font-size for the body, later?

good point though, this also provides a better performance on different devices and screensizes.

Reply:
You are right! At first I didn't like the "weird" numbers for the rem-values and was worried it wouldn't scale well. But now I replaced it with rem and it looks fine.